libs/scraper-archive/src/testData.ts

```
@@ -95,21 +114,34 @@ export class TestData {
+      }
+    }
+
+    // Clean up spaces in text, older test files did not clean
+    // before writing...
+    if (rawJson.data.text) {
+      rawJson.data.text = rawJson.data.text.replace(/\s+/g, " ").trim();
+    }
     return rawJson;
   }
 
   *elm_iter(fnName: string): Generator<TestElmData, void, unknown> {
```

ESLint

[error] Missing space after *.

(generator-star-spacing)